lib/main/atom/tooltipManager.ts

```diff
@@ -15,10 +15,19 @@ import escape = require("escape-html")
 
 export function getFromShadowDom(element: JQuery, selector: string): JQuery {
   var el = element[0]
-  var found = (<any>el).rootElement.querySelectorAll(selector)
+  var found = (<any>el).querySelectorAll(selector)
```

Comment:
It's fine, I'm not here to judge, I just pointed it out since it was a simple request.

This package should get some serious maintainers...